arm-containerregistry/2017-03-01/examples/RegistryCreate.json

```diff
@@ -0,0 +1,51 @@
+{
+    "parameters": {
+        "api-version": "2017-03-01",
+        "subscriptionId": "subid",
+        "resourceGroupName": "rgname",
+        "registryName": "registry",
+        "registryCreateParameters": {
+            "location": "westus",
+            "tags": {
+                "key": "value"
+            },
+            "sku": {
+                "name": "Basic"
+            },
+            "properties": {
+                "adminUserEnabled": true,
+                "storageAccount": {
+                    "id": "/subscriptions/subid/resourceGroups/rgname/providers/Microsoft.Storage/storageAccounts/storageaccount",
```

Comment:
Better to put an empty guid for subid, tools can take assumption that subId is guid